crates/bevy_mesh/src/index.rs

```diff
@@ -135,3 +155,28 @@ impl From<&Indices> for IndexFormat {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::Indices;
+    use wgpu::IndexFormat;
+
+    #[test]
+    fn test_indices_push() {
```

Comment:
Can you add a test case or test for the u16 -> u32 behavior, since that's particularly important to get right?

Reply:
The test is there. Code below:

```
        indices.push(0x10000);
        assert_eq!(IndexFormat::Uint32, IndexFormat::from(&indices));
        assert_eq!(vec![10, 0x10000], indices.iter().collect::<Vec<_>>());
```

converts `Indices` from `U16` to `U32`. I'll add a comment to make it more clear.